script.js

```diff
@@ -1,14 +1,22 @@
+function showModal() {
+  document.getElementById('exampleDialog').showModal();
+}
```

Comment:
- [ ] 👏 Good job on implementing the popup functionality! However, I noticed that the popup content is not dynamically generated for each project. As per the project requirements, the popup should display details specific to the project that the user clicks on. To address this, please update your code to dynamically generate the popup content based on the project that the user interacts with. This will ensure that each project's details are displayed correctly in the popup.

By making this improvement, your project will provide a more interactive and user-friendly experience, enhancing its overall functionality and usability. Keep up the good work! 🌟📝🚀